src/orng.cpp

```diff
@@ -70,6 +71,41 @@ ACTION orng::version() {
         config_table.emplace(get_self(), update_version_fn);
 }
 
+ACTION orng::setbwpayer(const eosio::name& payee, const eosio::name& payer) {
+    check(!is_paused(), "Contract is paused");
+    require_auth(payee);
+
+    auto it = bwpayers_table.find(payee.value);
+
+    if (it == bwpayers_table.end()) {
+        bwpayers_table.emplace(payee, [&](auto& rec) {
+            rec.payee = payee;
+            rec.payer = payer;
+            rec.accepted = false;
+        });
+    } else {
+        check(it->payer != payer, "payer for this contract has already set with that account");
+        bwpayers_table.modify(it, same_payer, [&](auto& rec) {
+            rec.payer = payer;
+            rec.accepted = false;
+        });
+    }
+}
+
+ACTION orng::acceptbwpay(const eosio::name& payee, const eosio::name& payer, bool accepted) {
+    check(!is_paused(), "Contract is paused");
+    require_auth(payer);
+
+    auto it = bwpayers_table.find(payee.value);
```

Comment:
I think we can shorten this to `require_find`, and the text below should probably be `payee does not exist`

Reply:
Resolved